### PR TITLE
Release readiness for Media 3.0.0

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   import com.adobe.marketing.mobile.Media;
+   ```
+
+   ### Kotlin
+
+   ```kotlin
+   import com.adobe.marketing.mobile.MobileCore
+   import com.adobe.marketing.mobile.Identity
+   import com.adobe.marketing.mobile.Analytics
+   import com.adobe.marketing.mobile.Media;
```
Comment (addb) [inline Documentation/getting-started.md]: remove `;`

File: README.md [hunk 2]
```diff
@@ -6,29 +6,29 @@
...
-- [Analytics](https://developer.adobe.com/client-sdks/documentation/adobe-analytics)
+Refer to the [getting started](./Documentation/getting-started.md) guide for setting up and using the SDK with your application.
 
-## Installing the AEP Media SDK for Android
 
-The AEP SDK supports Android API 19 (Kitkat) and newer.
+## Development
 
-Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app.  In your `build.gradle` file, include the latest version of following dependencies:
+**Open the project**
```
Comment (addb) [inline README.md]: nit: ### Open the project

File: README.md [hunk 3]
```diff
@@ -6,29 +6,29 @@
...
-```gradle
-implementation 'com.adobe.marketing.mobile:core:2.x.x'
-implementation 'com.adobe.marketing.mobile:identity:2.x.x'
-implementation 'com.adobe.marketing.mobile:analytics:2.x.x'
-implementation 'com.adobe.marketing.mobile:media:3.x.x'
-```
+To open and run the project, open the `code/build.gradle` file in Android Studio
 
-## Development
+**Run test application**
```
Comment (addb) [inline README.md:21]: nit: ### Run test application

Comment (addb) [reply README.md:21]: adding as headers gives us link to share

File: Documentation/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+## Add Media extension to your app
+
+1. Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. Add the Mobile Core, Identity, Analytics, and Media extensions to your project using the app's Gradle file:
+
+
+   ```
+   implementation 'com.adobe.marketing.mobile:core:2.+'
+   implementation 'com.adobe.marketing.mobile:identity:2.+'
+   implementation 'com.adobe.marketing.mobile:analytics:2.+'
+   implementation 'com.adobe.marketing.mobile:media:2.+'
```
Comment (addb) [inline Documentation/getting-started.md]: `...media:3.+`

File: code/media/build.gradle [hunk 5]
```diff
@@ -52,7 +52,7 @@ android {
 
 dependencies {
     //noinspection GradleDependency
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
+    implementation "com.adobe.marketing.mobile:core:$mavenCoreVersion"
     implementation 'androidx.annotation:annotation:1.3.0'
```
Comment (addb) [inline code/media/build.gradle]: aren't we using annotation:1.0.0 everywhere?

**Comment by addb** [comment]
@praveek Also update contributing.md and code of conduct. 
refer: 
https://github.com/adobe/aepsdk-edge-android/blob/main/.github/CONTRIBUTING.md
https://github.com/adobe/aepsdk-edge-android/blob/main/CODE_OF_CONDUCT.md

File: code/media/build.gradle (hunk 5 above)
Comment (addb) [reply code/media/build.gradle]: https://github.com/adobe/aepsdk-edge-android/blob/main/code/gradle.properties#L33
https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/build.gradle#L239

**Comment by addb** [comment]
Also should we rename dev branch to dev-3.0.0 and then merge to staging->main?

File: code/testapp/build.gradle [hunk 6]
```diff
@@ -48,14 +48,8 @@ dependencies {
...
-    implementation ('com.adobe.marketing.mobile:identity:2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation ('com.adobe.marketing.mobile:analytics:2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation ('com.adobe.marketing.mobile:assurance:2.0.0-SNAPSHOT'){
-        transitive = false
-    }
+    implementation 'com.adobe.marketing.mobile:core:2.+'
```
Comment (addb) [inline code/testapp/build.gradle:51]: Should we use minSDK on L11 as 19 or get al these values from rootproject like [this](https://github.com/adobe/aepsdk-edge-android/blob/main/code/app/build.gradle)?

**Comment by praveek** [comment]
> Also should we rename dev branch to dev-3.0.0 and then merge to staging->main?

It should be ok as we are done with all the dev work.

**Review by addb — APPROVED** [review]
lgtm!

nit: License header everywhere in scripts.

**Pull request merged**